web/src/test/java/org/openmrs/web/test/WebModuleActivatorTest.java

```diff
@@ -289,7 +291,8 @@ public void initializeInDatabase() throws Exception {
 	
 	@AfterEach
 	public void removeCurrentTransactionContext() throws Exception {
-		Class<?> clazz = OpenmrsClassLoader.getInstance().loadClass("org.springframework.test.context.transaction.TransactionContextHolder");
+		Class<?> clazz = OpenmrsClassLoader.getInstance()
```

Comment:
add this on the same line

Reply:
Hi Herbert, 
Thank you for your response. I am a bit confused on the comments that you made and was wondering if you could elaborate on the changes that you are requesting. I think my confusion lies in:
1) I do not believe I changed, or meant to change, these lines of code. 
2) instead of 

`Class<?> clazz = OpenmrsClassLoader.getInstance() `

do you want me to put

`Class<?> clazz = OpenmrsClassLoader.getInstance().loadClass("org.springframework.test.context.transaction.TransactionContextHolder");`

Reply:
I assume the line is to long and the formatter therefore split it into 2 lines which is ok from my side.